test/unit/bson_import.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { expect } = require('chai');
+const BSON = require('../../src/bson');
+
+describe('BSON Library Import', function () {
```

Comment:
we discussed organizing the tests here a little differently: using require.resolve instead of require, and splitting the tests into 2 describe blocks - one for bson-ext and one for js-bson, each of which should have one it block for 'should import the correct library' and another it block for 'should correctly roundtrip supported data types'; using helpers and/or a data table structure should help minimize code duplication